packages/core/src/event-bus/events/order-event.ts

```diff
@@ -1,6 +1,11 @@
-import { RequestContext } from '../../api/common/request-context';
-import { Order } from '../../entity';
-import { VendureEvent } from '../vendure-event';
+import { ModifyOrderInput } from '@vendure/common/lib/generated-types';
+import { ID } from '@vendure/common/lib/shared-types';
+
+import { RequestContext } from '../../api';
+import { Customer, Order } from '../../entity';
+import { VendureEntityEvent } from '../vendure-entity-event';
+
+type OrderInputTypes = Order | Customer | ModifyOrderInput | ID | { customFields: any };
```

Comment:
Actually we don't need `Order` or `ID` as input types here. I realized this is redundant info anyway when you delete an order, because you already get the deleted order as the main entity payload.